src/aavrutiDJun20/PrimeNumbers.java

```diff
@@ -0,0 +1,47 @@
+package aavrutiDJun20;
+
+public class PrimeNumbers {
+	
+	//Print whether given number is Prime Number or not
+	void primeNumberOrNot(int number) {
+		boolean flag = false;
+		for(int index=2;index<number/2;index++) {
+			if(number%index==0)
+				flag = true;
+		}
+		if(flag==true)
+			System.out.println(number + " is not a Prime Number");
+		else
+			System.out.println(number + " is a Prime Number");		
+	}
+	
+	//Calculate Sum, Count and Average of Prime Numbers between 2 - 100
+	void primeNumber2To100() {
```

Comment:
@AavrutiDJun20 it is better to take parameter, like startNum and endNum , so that tomorrow if range change to 2 to 200 you don't need to update your loop.

Reply:
Done